_docs/browser-certificates.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: Browser Certificates
+permalink: /docs/browser-certificates/
+---
+
+### Firefox
+
+The Firefox driver properly handles untrusted certificates by default.
+
+If you have a trusted certificate, but there is some other certificate error such as a hostname mismatch (eg. using a production certificate in test), you should do the following:
+
+{% highlight ruby %}
+profile = Selenium::WebDriver::Firefox::Profile.new
+profile.assume_untrusted_certificate_issuer = false
+b = Watir::Browser.new :firefox, :profile => profile
+{% endhighlight %}
+
+The reason this is needed is explained [here](http://code.google.com/p/selenium/wiki/UntrustedSSLCertificates#Implementation_details).
```

Comment:
Selenium is no longer on Google Code, so the link should be https://github.com/SeleniumHQ/selenium/wiki/Untrusted-SSL-Certificates#implementation-details